docs/staking-and-delegation/managing-stake-tao-app.md

```diff
@@ -2,13 +2,33 @@
 title: "Managing Stake with tao.app"
 ---
 
-[TAO.app](https://tao.app) offers a portal to real-time tokenomic data about the Bittensor ecoystems and all its subnets, and allows users to securely manage their stake using the Bittensor wallet browser extension.
+[Tao.app](https://tao.app) offers a portal to real-time tokenomic data about the Bittensor ecoystems and all its subnets, and allows users to securely manage their stake using the Bittensor wallet browser extension.
+
+At the very top of the main page, there are several metrics which gives some of the most important information an ecosystem user would likely want to be informed with:
+
+* **TAO Price**: The current price of the TAO coin and percentage increase or decrease over the last day.
+* **1D Volume**: The total amount of TAO traded over the last day and the percentage change in this, increase or decrease, over the lst 24 hour period.
```

Comment:
* **1D Volume**: The total amount of TAO in US Dollar traded over the last day across major exchanges and its percentage change over the last 24 hour period. (Source: coinmarketcap)